formSchemas.js

```diff
@@ -18,9 +18,27 @@ const loginSchema = {
       options: [API.getMatches()],
       errorMessage: 'errors.login.code',
     },
+  }
+}
+const sinSchema = {
+  sin: {
+    customSanitizer: {
+      options: value => {
+        //We want to remove any spaces, dash or underscores
+        return value ? value.replace(/[ \-_]*/g, '') : value
```

Comment:
Is there a way to pass non-string values back as post parameters? 

I would kind of think we're okay, but maybe see what an empty value or a single digit number end up as (probably a good test).